var/spack/repos/builtin/packages/lcov/package.py

```diff
@@ -14,10 +14,12 @@ class Lcov(MakefilePackage):
 
     homepage = "http://ltp.sourceforge.net/coverage/lcov.php"
     url = "https://github.com/linux-test-project/lcov/releases/download/v2.0/lcov-2.0.tar.gz"
+    git = "https://github.com/linux-test-project/lcov.git"
     maintainers("KineticTheory")
```

Comment:
```suggestion
    homepage = "http://ltp.sourceforge.net/coverage/lcov.php"
    url = "https://github.com/linux-test-project/lcov/releases/download/v2.0/lcov-2.0.tar.gz"
    git = "https://github.com/linux-test-project/lcov.git"
    
    maintainers("KineticTheory")
```

Quick suggestion to add a space between the variables and `maintainers()` directive. Otherwise this looks good to me.